src/components/LandingPage/LandingPage.jsx

```diff
@@ -70,4 +81,14 @@ export const LandingPage = () => {
         </Container>
     </React.Fragment>
     )
-}
\ No newline at end of file
+}
+
+function mapDispatchToProps(dispatch) {
```

Comment:
Considering using new React-Redux hooks instead of slightly older method (map***toProps) here.